### DISCUSSION TITLE
how to run at a local server

### DISCUSSION THREAD

**Discussion by atasoyk** [subins2000/WebDrop#45] [General]
hello, couldn't find a way to run the files on windows for a local server without internet. is there a exe file for this? tnx

**Comment by subins2000** [comment]
To run on a local network, follow these steps
1. Run a WebTorrent tracker on a server : https://github.com/webtorrent/bittorrent-tracker (this host should be accessible to anyone in the network)
2. Clone this repository and add the tracker's hostname created in step 1 to announceURLs : https://github.com/subins2000/WebDrop/blob/master/src/main.ts#L53
3. Build WebDrop and host it in the same server as tracker or somewhere else.

Now people in the network can open a browser and visit WebDrop
